LICENSE

```diff
@@ -1,21 +1 @@
-The MIT License (MIT)
-
-Copyright (c) 2016 Onfido
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
+Please see the terms of your Onfido Services Agreement which you are bound by with respect to your use of the Onfido SDK. To the extent the Onfido SDK includes copies or substantial portions of software licenced under third party terms and conditions this will be set out at https://github.com/onfido/onfido-sdk-ui/blob/master/dist/onfido.LICENSES.txt and https://github.com/onfido/onfido-sdk-ui/blob/master/dist/demo.LICENSES.txt. In the event of any inconsistency between the Onfido Services Agreement and any third party terms and conditions, the third party terms and conditions shall take precedence over the rights and restrictions granted in the Onfido Services Agreement solely in respect of such third party dependencies set out at https://github.com/onfido/onfido-sdk-ui/blob/master/dist/onfido.LICENSES.txt and https://github.com/onfido/onfido-sdk-ui/blob/master/dist/demo.LICENSES.txt.
```

Comment:
At the link below you will only find the license comments that are included in third-party dependencies codebases.
https://github.com/onfido/onfido-sdk-ui/blob/master/dist/onfido.LICENSES.txt
https://github.com/onfido/onfido-sdk-ui/blob/master/dist/demo.LICENSES.txt

The full list of third party licenses can be found here https://github.com/onfido/onfido-sdk-ui/blob/master/licenses.json.

Maybe we should create a folder and place all the files related to licenses there.

Reply:
Yeah, looks like it'll be better to consolidate everything in a folder. Currently the comments get generated in `dist` while the full list is in the root folder. I'll raise a ticket to address that.

Reply:
As discussed created a separate ticket for consolidating the files.